src/test/determinism/determinism1.test.shadow.config.yaml

```diff
@@ -21,9 +21,36 @@ network:
         ]
       ]
 hosts:
-  testnode:
+  testnode1: &host
     network_node_id: 0
-    quantity: 10
+    ip_addr: 11.0.0.1
```

Comment:
OOC why does this need explicit IPs now? Or maybe this is  an orthogonal improvement to not implicitly rely on shadow's auto IP assignment in this test?

Reply:
Previously with a `quantity`, show would create hosts `hostname{x}` for x in 0..10 and assigns each host an increasing IP address. So "hostname1" is assigned "11.0.0.1", "hostname2" is assigned "11.0.0.2", etc.

Without the `quantity` option, Shadow iterates over hosts in alphabetical order and assigns them an increasing IP address. But the hostname alphabetical order is "hostname1", "hostname10", "hostname2", "hostname3", etc. So "hostname1" is assigned "11.0.0.1", "hostname10" is assigned "11.0.0.2", "hostname2" is assigned "11.0.0.3", etc.

Reply:
oh! I see. And I guess the test actually use the IP addresses instead of doing name resolution?

Reply:
The simulation itself is fine I think, but the verification scripts after aren't. They do something along the lines of:

```
diff a/hosts/hostname{X}/11.0.0.{X}.pcap b/hosts/hostname{X}/11.0.0.{X}.pcap
```

Reply:
Gotcha; an explanatory comment in the config files might be helpful in the future

Reply:
Added some comments.